### PR TITLE
Bitcoin & LND data migration: added -og flag to rsync command + LND chown command

### LINKED ISSUE #933: [FEATURE REQ]  update your rsync command to include maintaining owner and group
### What & why

When following [this guide for transferring data from an on old install](https://raspibolt.org/faq.html#can-i-update-my-raspibolt-2-to-the-new-version), you use the rsync command with the flags 

-rhvP

You should update this to use the flags

-rhvPog

So that owner and group are maintained as well as permissions. 

Either that or you need to add the commands to update owner and group, ie

chown -r bitcoin:bitcoin /data/bitcoin

etc.

### PR DESCRIPTION
**Pull request by VajraOfIndra**
#### What

See discussion in #933. The PR adds presently missing guidelines concerning the ownership of the Bitcoin and LND data folders when migrating from v2 to v3.

### Why

To improve the change of users successfully migrating their data to a new node

#### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [x] simple bug fix

Fixes #933 

copy

### CODE REVIEW

**Review by visskiss — APPROVED** [review]
Looks good.

**Review by visskiss — APPROVED** [review]

File: faq.md [hunk 1]
```diff
@@ -86,7 +86,8 @@ Once you set up your new RaspiBolt 3, restore your old LND node setup.
   $ sudo mkdir /mnt/thumbdrive
   $ lsblk -pli
   $ sudo mount /dev/sdb1 /mnt/thumbdrive/
-  $ sudo rsync -rhvP --append-verify /mnt/thumbdrive/lnd /data
+  $ sudo rsync -rhvPog --append-verify /mnt/thumbdrive/lnd /data
+  $ sudo chown -r lnd:lnd /data/lnd
```
Comment (Stadicus) [inline faq.md:90]: I think the recursive attribute must be uppercase: `chown -R`

```sh
$ chown --help
  -R, --recursive        operate on files and directories recursively
```

**Review by Stadicus — CHANGES_REQUESTED** [review]
One small comment, otherwise LGTM

File: faq.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Copy it to a USB thumbdrive, so that you can restore it later.
 * Copy the whole LND data directory to the thumbdrive
 
   ```sh
-  $ sudo rsync -rhvP --append-verify /mnt/ext/lnd /mnt/thumbdrive/
+  $ sudo rsync -rhvPog --append-verify /mnt/ext/lnd /mnt/thumbdrive/
```
Comment (RooSoft) [inline faq.md:64]: Wondered what those new switches were about, so I'll paste them here

-o, --owner                 preserve owner (super-user only)
-g, --group                 preserve group

Seems to make a lot of sense...

**Review by RooSoft — CHANGES_REQUESTED** [review]
Aside from the `chown -r` which should be `chown -R`, it's all good to me

File: faq.md (hunk 1 above)
Comment (VajraOfIndra) [reply faq.md]: Fixed, PTAL

File: faq.md (hunk 2 above)
Comment (VajraOfIndra) [reply faq.md:64]: Fixed, PTAL

**Review by RooSoft — APPROVED** [review]
This is the way

**Review by Stadicus — APPROVED** [review]
LGTM

**Pull request merged**